### PR TITLE
Renames Loyalty implants to Mindshield implants

### PR DESCRIPTION
**Pull request by Kyep**
Exactly what it says on the tin.

Note: this change only affects player-visible references to loyalty implants. Not changing their names in the code to avoid map conflicts.

Reason: lots of ahelps like "does a loyalty implant make me loyal?"
No, it doesn't. It just prevents most forms of brainwashing.

:cl: Kyep
tweak: "Loyalty" implants are now known as "Mindshield" implants
/:cl:

### CODE REVIEW

**Comment by Fox-McCloud** [comment]
https://github.com/tgstation/tgstation/pull/18345

**Comment by Kyep** [comment]
OK, you posted a link, but what does the link mean?

Does it mean: port that instead?

Does it mean: it wasn't popular enough to pass there, don't bother?

Does it mean something else?

Just posting the link doesn't explain anything.

**Comment by KasparoVy** [comment]
cuz TG did this PR first and for the exact same reasons

**Comment by theColdflame** [comment]
I'd suggest the flavour text on implantation/removal be changed as well.

**Comment by Kyep** [comment]
@theColdflame that was changed, from "You feel a surge of loyalty towards Nanotrasen." to "You mind feels hardened somehow."

**Comment by Krausus** [comment]
There are still references to loyalty in mind.dm, shadowling_abilities.dm, and implant_mindshield.dm (the loyalty implanter and implant case).

I'd say they should probably be renamed from `loyalty` in the code, for the sake of consistency. There's only a few instances of them across the maps, so any conflicts it causes should be easy to fix.

The changelog entry should mention that this change is non-functional, since "loyalty" implants have functioned this way for quite some time.

Also, this is going to require quite a few changes to the wiki.

File: code/game/objects/items/weapons/implants/implant_mindshield.dm [hunk 1]
```diff
@@ -29,7 +29,7 @@
 		if(target.mind in ticker.mode.cult)
 			to_chat(target, "<span class='warning'>You feel the corporate tendrils of Nanotrasen try to invade your mind!</span>")
 		else
-			to_chat(target, "<span class='notice'>You feel a surge of loyalty towards Nanotrasen.</span>")
+			to_chat(target, "<span class='notice'>You mind feels hardened somehow.</span>")
```
Comment (Krausus) [inline code/game/objects/items/weapons/implants/implant_mindshield.dm]: > Your mind

File: code/modules/mob/living/carbon/superheroes.dm [hunk 2]
```diff
@@ -191,7 +191,7 @@
 					if(isloyal(target))
 						to_chat(usr, "<span class='notice'>They are enslaved by Nanotrasen. You feel their interest in your cause wane and disappear.</span>")
 						usr.visible_message("<span class='danger'>[usr] stops talking for a moment, then moves back away from [target].</span>")
-						to_chat(target, "<span class='danger'>Your loyalty implant activates and a sharp pain reminds you of your loyalties to Nanotrasen.</span>")
+						to_chat(target, "<span class='danger'>Your mindshield implant activates and a sharp pain reminds you of your loyalties to Nanotrasen.</span>")
```
Comment (Krausus) [inline code/modules/mob/living/carbon/superheroes.dm]: This message probably shouldn't imply that the implant is forcing you to be loyal.

**Comment by taukausanake** [comment]
Wouldn't grepping all the files for "/obj/item/weapon/implant/loyalty" into new name solve possible conflicts in the maps? And maybe all instances of "loyalty implant" too? Just a thought

**Comment by Kyep** [comment]
@krausus text changed. 

I am not going to do a search/replace on /obj/item/weapon/implant/loyalty for /obj/item/weapon/implant/mindshield while there are 3 map merge PRs pending. 

The point of this PR is to change the way they appear to players - while changing as little of the code as possible - and not risking any conflicts.

**Comment by Krausus** [comment]
Almost there...
- shadowling_abilities.dm, lines 345 & 347
- implant_mindshield.dm, line 55

I guess changing the paths in the code can wait... though it's odd to have renamed implant_loyalty.dm but none of the paths...

**Comment by Fox-McCloud** [comment]
Reason I linked the PR is because that's where this idea originally came from and to provide you with a an easier way to locate all of what you would have to change and what to change it to for better clarity.

**Comment by TullyBurnalot** [comment]
All for this. It has been coming for a long time.

**Comment by Kyep** [comment]
@Krausus changed those lines

File: code/modules/mob/living/carbon/alien/alien.dm [hunk 3]
```diff
@@ -217,7 +217,6 @@
 		if(judgebot.check_for_weapons(r_hand))
 			threatcount += 4
 
-	//Loyalty implants imply trustworthyness
```
Comment (tigercat2000) [inline code/modules/mob/living/carbon/alien/alien.dm]: leave the comment

File: code/game/objects/items/weapons/implants/implant_mindshield.dm [hunk 4]
```diff
@@ -1,18 +1,18 @@
...
 	var/dat = {"<b>Implant Specifications:</b><BR>
 				<b>Name:</b> Nanotrasen Employee Management Implant<BR>
 				<b>Life:</b> Ten years.<BR>
-				<b>Important Notes:</b> Personnel injected with this device tend to be much more loyal to the company.<BR>
+				<b>Important Notes:</b> Personnel injected with this device can better resist mental compulsions.<BR>
 				<HR>
 				<b>Implant Details:</b><BR>
 				<b>Function:</b> Contains a small pod of nanobots that manipulate the host's mental functions.<BR>
-				<b>Special Features:</b> Will prevent and cure most forms of brainwashing.<BR>
+				<b>Special Features:</b> Will prevent most forms of brainwashing.<BR>
```
Comment (tigercat2000) [inline code/game/objects/items/weapons/implants/implant_mindshield.dm]: pretty sure they still deconvert revs

**Comment by HereticChurch** [comment]
I remember suggesting this way back when we first changed the function of loyalty implants to forego the whole loyalty thing. Glad we've finally come around to my way of thinking. :+1:

File: code/modules/mob/living/carbon/alien/alien.dm (hunk 3 above)
Comment (Kyep) [inline code/modules/mob/living/carbon/alien/alien.dm]: Changed to mindshield.

**Comment by Kyep** [comment]
@Fox-McCloud @TheDZD ready for review.

**Comment by Kevak** [comment]
I approve.

**Pull request merged**